src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.*;
+import core.basesyntax.service.impl.*;
+import core.basesyntax.strategy.*;
+import core.basesyntax.strategy.impl.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        String readFilePath = "reportToRead.csv";
+        String writeFilePath = "writtenReport.csv";
+
+        FileReader fileReader = new FileReaderImpl();
```

Comment:
Consider using the try-with-resources statement to ensure that the resources are closed properly after use.